src/world/location.lua

```diff
@@ -0,0 +1,51 @@
+local Location = {}
+
+function Location.bodyCenter3(body)
+	local x, y = body:getPosition()
+	local a = body:getAngle()
+	return x, y, a
+end
+
+function Location.bodyCenter6(body)
```

Comment:
It's not clear what the 3 and 6 mean. Could you call it `bodyCenterAndVelocity` or something?

Reply:
We decided that since my suggested name is too long to fit in under 80 columns in the common places where it's used, and since we don't have any better ideas, we will just leave it as for now and come back to it later.